presqt/api_v1/urls.py

```diff
@@ -1,8 +1,10 @@
 from django.urls import path
 
 from presqt.api_v1 import api_root
-from presqt.api_v1.views.resource import ResourceCollection, Resource, ResourceDownload
-from presqt.api_v1.views.target import TargetCollection, Target
+from presqt.api_v1.views.resource.resource import Resource
```

Comment:
I think we've talked before how you could import these views into the first `resource` package and then import all from there to clean this up a bit... feel free to ignore.